src/applications/vaos/project-cheetah/components/ScheduleNewProjectCheetah.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function ScheduleNewProjectCheetah({ startNewAppointmentFlow }) {
```

Comment:
This should live in the appointments folder, rather than project-cheetah, since it's only used in that section of the app. Similar to these sections for regular appointments and express care.